packages/ui/src/components/sheet/components/sheet-header.native.tsx

```diff
@@ -19,11 +19,13 @@ export function SheetHeader({ title, icon, onPressSupport }: SheetHeaderProps) {
           </Text>
         </Box>
       </Box>
-      <Box alignItems="flex-end" flex={1}>
-        <TouchableOpacity onPress={onPressSupport} zIndex="10">
-          <QuestionCircleIcon variant="small" />
-        </TouchableOpacity>
-      </Box>
+      {onPressSupport && (
```

Comment:
Sometimes we were showing a `?` icon when there was no link so I have changed that